src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -463,14 +463,12 @@ public static void uncaughtException(Thread t, Throwable e)
             {
                 if (e2 != e) // make sure FSError gets logged exactly once.
                     logger.error("Exception in thread " + t, e2);
-                FileUtils.handleFSError((FSError) e2);
```

Comment:
since this is now handled by `JVMStabilityInspector.inspectThrowable` the logging should also be handled there.  Also, inspect is recursive so we don't need to walk the cause change anymore